src/FunderMaps.WebApi/DataTransferObjects/AddressDto.cs

```diff
@@ -30,23 +31,5 @@ public sealed class AddressDto
         /// </summary>
         [Required(AllowEmptyStrings = false)]
         public string Street { get; set; }
-
```

Comment:
Why was this removed?

Reply:
It exposes internals to unauthorized users